Formula/c/cairo.rb

```diff
@@ -27,11 +26,11 @@ class Cairo < Formula
 
   head do
     url "https://gitlab.freedesktop.org/cairo/cairo.git"
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
   end
 
+  depends_on "cmake" => :build
```

Comment:
Is `cmake` needed?

Reply:
Added as requested when building with CI.

ref: https://github.com/Homebrew/homebrew-core/actions/runs/6287881931/job/17072824117#step:3:234

Reply:
The CI run you linked to was on ik11235/homebrew-core@5272bdff85e25f97d00bb3bb87921ec90904ec14. The root cause was that `pixman` could not be found with `pkg-config` (see [here](https://github.com/Homebrew/homebrew-core/actions/runs/6287881931/job/17072824117#step:3:83)), because the `pkg-config` dependency was removed in that commit.

Reply:
> CMake was only used by Meson as a fallback. We have pkg-config, so we don't need CMake here.

Thanks, fixed.

https://github.com/Homebrew/homebrew-core/pull/145154/commits/d1b10e8c6eb8345afa36c267779e4fba0ff75050